src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball[] balls = {new Lottery().getRandomBall(),
+                new Lottery().getRandomBall(),
+                new Lottery().getRandomBall()};
```

Comment:
let's create a separate variable for `Lottery` instance and use it:
```
Lottery lottery = new Lottery();
```
and better call method `getRandomBall()` in the loop